grakn-core/src/main/java/ai/grakn/concept/Rule.java

```diff
@@ -84,34 +71,56 @@
     @CheckReturnValue
     Stream<Type> getConclusionTypes();
 
-    //---- Inherited Methods
+    //------------------------------------- Modifiers ----------------------------------
+    /**
+     * Changes the {@link Label} of this {@link Concept} to a new one.
+     * @param label The new {@link Label}.
+     * @return The {@link Concept} itself
+     */
+    Rule setLabel(Label label);
+
     /**
      *
-     * @return The type of this Graql
+     * @return The super type of this {@link Rule}
```

Comment:
fun pedantry: we shouldn't say "super type". I've been going with "super concept", but you can just say "super" if you want.

Reply:
🍌